lib/list.gd

```diff
@@ -2376,36 +2376,21 @@ DeclareGlobalFunction( "ListWithIdenticalEntries" );
 ##
 #F  PlainListCopy( <list> ) . . . . . . . .  make a plain list copy of a list
 ##
+##  <#GAPDoc Label="PlainListCopy">
 ##  <ManSection>
 ##  <Func Name="PlainListCopy" Arg='list'/>
 ##
 ##  <Description>
+##  This function returns a list equal to its argument, in a plain list
+##  representation.
 ##  This is intended for use in certain rare situations,
-##  such as before objectifying.
-##  Normally, <C>ConstantAccessTimeList</C> should be enough.
```

Comment:
This exists and was a typo for ConstantTimeAccessList

Reply:
That said, I don't think this sentence was helpful, so I am fine with removing it :-)